app/assets/stylesheets/bootstrap-overrides.scss

```diff
@@ -0,0 +1,31 @@
+/* Bootstrap overrides go here. This is included after bootstrap. */
+
+/* 979px - 980px is where nav-collapse gets triggered */
+@media (max-width: 979px) {
```

Comment:
Is there a way we can make the breakpoint feature flaggable? Possibly with a class on body? On desktop, the breakpoint is still getting applied at thinner widths and triggering the hamburger menu. I think we might need some of the hidden/visible.

I'd like us to get a little further with some of the other changes as well before these updates become visible for the schools.